.github/workflows/release-pypi.yaml

```diff
@@ -0,0 +1,36 @@
+name: Release Python Package
```

Comment:
I've standardised all the workflows so far I'm working on just to say "Release X" where X is the name of the language. The filename can stay the same.

This way in large polyglots I know exactly what's being released